boot/makebootable.go

```diff
@@ -211,6 +211,7 @@ func makeBootable20RunMode(model *asserts.Model, rootdir string, bootWith *Boota
 		Mode:           "run",
 		RecoverySystem: filepath.Base(bootWith.RecoverySystemDir),
 		Base:           filepath.Base(bootWith.BasePath),
+		CurrentKernels: []string{filepath.Base(bootWith.Kernel.MountFile())},
```

Comment:
can use Filename now?

Reply:
ah yes I missed that one too in rebasing good catch

Reply:
There's quite a few more that should be changed over, but I will do that in an orthogonal PR

Reply:
https://github.com/snapcore/snapd/pull/8144